feature/policy_forwarding/otg_tests/mpls_gre_udp_macsec/README.md

```diff
@@ -57,7 +56,7 @@ Verify:
 * No packet loss while forwarding at line rate
 * Traffic equally load-balanced across bundle interfaces in both directions
 * Header fields are as expected in both directions
-* Traffic is dropped (100 percent) when the must-secure MACSec sessions are down by disabling MACsec on ATE ports
+* Traffic is dropped (100 percent) when the must-secure MACSec sessions are down by changing a key on one side to a mismatch & forcing renegotiation on ATE ports
```

Comment:
This comment doesn't take care of a scenario where due to connection issues - MKA session goes down. I think the comment should be generic to say that if there are no successful MKA session on the port, then all the traffic will be dropped.